mmv1/third_party/terraform/go.mod.erb

```diff
@@ -5,7 +5,7 @@ go 1.18
 
 require (
 	cloud.google.com/go/bigtable v1.16.0
-	github.com/GoogleCloudPlatform/declarative-resource-client-library v1.19.0
+	github.com/GoogleCloudPlatform/declarative-resource-client-library v1.25.0
```

Comment:
Note: DCL upgrades have been having some difficulties, and https://github.com/GoogleCloudPlatform/magic-modules/pull/6639 is the current upgrade PR. We'd prefer to avoid mixing resource contributions w/ upgrade PRs, as having multiple focus areas in a PR can get messy. That's going to mean we're blocked on https://github.com/GoogleCloudPlatform/magic-modules/pull/6639 merging for this change.

Reply:
ack, should I keep my upgrade changes then or remove them from this PR? (I don't know if we can add #6639  as a dependency for this PR somehow)